lisa/tools/ip.py

```diff
@@ -336,6 +336,70 @@ def get_default_route_info(self) -> tuple[str, str]:
         assert_that(dev_match.groups()).is_length(1)
         return dev_match.group(1), dev_match.group()
 
+    def add_route_to(self, dest: str, via: str, dev: str) -> None:
```

Comment:
There's two routing tables, the azure level one governing what traffic is dropped in the virtual network and the linux kernel level one governing what traffic is dropped before it's sent out over the NIC.
 
In the test we modify the az routing table for two subnets on the test VM and set up rules to forward all traffic on those subnets to the l3fwd VM. We're using the az routing table to create a network virtual applicance essentially, with the goal of having the l3fwd VM bridge the second and third subnet in the test environment.

You need the AZ routing table to tell Azure not to drop the traffic as it transits between nodes, we're letting azure know it's ok to send traffic to this forwarder even though it's addressed to another IP address.

When that's done, we also need to use ip to modify the linux kernel routing tables in the guests to tell them to forward traffic destined for the sender/receiver to the forwarder first. 

Since LISA creates three subnets when you create a VM with three nics, the l3fwd test has to do some work to disable the extra NICs that might leak traffic on the sender and receiver VMs. That work is done with these ip route functions. 

We remove some routing rules and add others to tell the linux kernel it's OK to route traffic bound for the sender or receiver IP to the l3fwd VM over a specific interface.